### PR TITLE
Feature/order list version2.0

### PR DESCRIPTION
**Pull request by Tommybui28**
Please help, orders not showing for /driver

### CODE REVIEW

**Review by ney601 — APPROVED** [review]
need to fix rendering issue

**Review by Jean-005 — APPROVED** [review]
Amazinggg

File: capstone_frontend/src/containers/DeliveryContainer.jsx [hunk 1]
```diff
@@ -110,7 +110,13 @@ const DeliveryContainer = () => {
                 {
                     path: "/driver",
                     element:
-                        <RouteDisplay />
+                        <>
```
Comment (Aebel-Shajan) [inline capstone_frontend/src/containers/DeliveryContainer.jsx:113]: Nice we got ordersss wooooo

**Review by Aebel-Shajan — APPROVED** [review]
lgtm

**Review by sahilpatel1906 — APPROVED** [review]

**Pull request merged**